components/authz-service/server/v2/project.go

```diff
@@ -113,7 +113,7 @@ func (s *ProjectState) GetProject(ctx context.Context,
 
 func (s *ProjectState) CreateProject(ctx context.Context,
 	req *api.CreateProjectReq) (*api.CreateProjectResp, error) {
-	p, err := storage.NewProject(req.Id, req.Name, storage.Custom)
+	p, err := storage.NewProject(req.Id, req.Name, storage.Custom, 0)
```

Comment:
:x: Can we use one of [the enum constant](https://github.com/chef/automate/blob/76a491ab2d2a7b7d8829e06a00d1b0593a7df803/components/authz-service/storage/v2/project_rules_status.go#L7-L20) here?
```suggestion
	p, err := storage.NewProject(req.Id, req.Name, storage.Custom, storage.RulesStatusError)
```
`0` is `RulesStatusError`. Makes me wonder if `NoRules` wouldn't me more appropriate...